tests/Common/ComponentsTest.php

```diff
@@ -2520,15 +2570,19 @@ public function testRowChangesAfterConfigurationCopy()
         // copy config version 2
         $copiedConfig = $components->createConfigurationFromVersion('wr-db', $config->getConfigurationId(), 2, 'test');
         $response = $components->getConfiguration('wr-db', $copiedConfig["id"]);
+        $this->assertSame('Copied from configuration "main-1" version 2', $response['changeDescription']);
```

Comment:
Nevím, jestli nepoužít `name` (`'name'`) místo `configurationId` (`'main-1'`)? V reálu tam budou nějaký nesmyslný čísla. Ztratíme sice přesnou vazbu, ale tu stejně na nic nepoužíváme a ze stringu bychom to neparsovali.

Reply:
Co `Copied from configuration "name" (main-1) version 2`?

Reply:
Jo, to je super.

Reply:
jo takhle to vypadá dobře.